packages/visualizations/src/components/utils/BackButton.svelte

```diff
@@ -0,0 +1,21 @@
+<script>
+</script>
+
+<button class="back-button" type="button" on:click>
```

Comment:
`on:click` is unused ? To delete ?

Reply:
Same as above, as far as I've tested it without all this on:clik it doesn't forward the on:click passed to the component I guess